vyxal/elements.py

```diff
@@ -449,6 +450,7 @@ def process_element(
         "lhs = pop(stack, 1, ctx); vy_type(lhs) == NUMBER_TYPE and time.sleep(lhs)",
         1,
     ),
+    "ø%": process_element("sha256(lhs.encode()).hexdigest()", 1),
```

Comment:
I'd prefer if this were implemented as a function with the `@element` decorator, something like
```python
@element("ø%", 1)
def hash(lhs, ctx):
    """Element ø%
    ...
    """
    if isinstance(lhs, str):
        return sha256(lhs.encode()).hexdigest()
    else:
        <raise some kind of error saying lhs must be a string>
```